### DISCUSSION TITLE
Missing DLLs

### DISCUSSION THREAD

**Discussion by kahlenberg** [pelicanmapping/osgearth#2501] [Support Q&A] [answered]
Hi,
I cloned the osgearth 3.5 with submodules and built on Windows-10 64-bit with vcpkg (bootstrap-vcpkg), then I set the PATH in environment.
However when I try to start an exe in build/bin I receive errors that some DLLs i.e. osgEarth.dll, are not found. Maybe I forgot to install some dependencies?
I also tried with GL3 (`set(osg_OPENGL_PROFILE GL3)`)

**Comment by gwaldron** [comment, marked as answer]
You probably have you path wrong, but if you're sure you don't, try running this tool to see what DLLs cannot be found:
https://dependencywalker.com/

**Reply by kahlenberg** [reply]
Thank you very much for a fast reply! I will try and report the result.
